### ISSUE TITLE
Can uploading of blog image be made optional rather than mandatory?

### ISSUE THREAD

**Issue by tejaskp114** [mrgoldy/ultimateblog#5]
Hi mrgoldy,

Thanks a lot for creating a wonderful blog mod for phpbb. Can uploading of blog image be made optional rather than mandatory under blog settings? Not every user wishes to upload image. would be of great help if this option is provided.

Thanks,
tejaskp114

**Comment by mrgoldy** [comment]
This will require a complete style rechange which I am not looking to do. I could add an option to select a 'default' image set by the administrator.

**Comment by tejaskp114** [comment]
thanks, this option would help a lot

**Comment by FlynnTeK** [comment]
I agree setting a default image is best you could do as Facebook does and use an outline image or google uses the persons initials for there default image
